.github/workflows/build-and-deploy.yml

```diff
@@ -0,0 +1,82 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:15
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_DB: postgres
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: testpassword
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'corretto'
+          java-version: '17'
+
+      - name: Start Tomcat
+        run: |
+          docker run -d --network=host --name tomcat tomcat:latest
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+        with:
+          gradle-version: release-candidate
+
+      - name: Create database table
+        run: |
+          psql -h localhost -U postgres -d postgres -f ./app/src/main/java/org/vss/impl/postgres/sql/v0_create_vss_db.sql
+        env:
+          PGPASSWORD: testpassword
+
+      - name: Build and Deploy
+        run: |
+          # Print Info
+          java -version
+          gradle --version
+
+          gradle wrapper --gradle-version 8.1.1
+          ./gradlew --version
+          ./gradlew build
+
+          docker cp app/build/libs/app-1.0.war tomcat:/usr/local/tomcat/webapps/vss.war
+
+      - name: Hit endpoint to verify service is up
+        run: |
+          # Wait for tomcat-startup
+          sleep 5
+
+          # Verify curl exits with error-code 0
```

Comment:
Isn't clear what this comment is referring to. I'd just drop it if a non-zero exit causes a failure.